elasticsearch-extensions/lib/elasticsearch/extensions/test/profiling.rb

```diff
@@ -53,7 +53,6 @@ module Profiling
         #
         def measure(name, options={}, &block)
           ___          = '-'*ANSI::Terminal.terminal_width
-          test_name    = self.name.split('::').last
```

Comment:
Ha, thanks for pointing this out! Actually, the problem is reversed -- the `test_name` variable tries to remove the namespaces from the test name, and it should be used later on line 62, instead of the original `name`.